src/microbe_stage/CellHexesPhotoBuilder.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using Godot;
+
+public class CellHexesPhotoBuilder : Spatial, IPhotographable
+{
+    private float radius;
+    private bool radiusDirty;
+    private MicrobeSpecies? species;
+
+    public string SceneToPhotographPath => "res://src/microbe_stage/CellHexesPhotoBuilder.tscn";
+
+    public float Radius
+    {
+        get
+        {
+            if (radiusDirty)
+                CalculateRadius();
+
+            return radius;
+        }
+    }
+
+    public MicrobeSpecies? Species
+    {
+        get => species;
+        set
+        {
+            species = value;
+            radiusDirty = true;
+        }
+    }
+
+    public void ApplySceneParameters(Spatial instancedScene)
```

Comment:
instanced sounds fine to me, this is probably a copy of what photographable classes have. Though now I'm a bit wondering why this is nor marked "override".

Reply:
Oh yeah, IPhotographable is an interface so of course this doesn't have the override keyword. This should definitely use the same parameter naming as the interface this implements.

Reply:
The interface also has the same naming it looks like. Notwithstanding, instance as a verb in the future engine versions would officially be instantiate so it would be less work to use this terminology now than later.

Though this is bike-shedding so I digress, I see you're fine with it so I guess both terms can be used interchangeably going forward.

Reply:
I personally always will end up writing instanced, it just sounds better to me but I'll also allow the other term to be used. If someone really wants to nitpick there might be some meaning difference but I'm not going to really care.